torch/_dynamo/variables/misc.py

```diff
@@ -758,7 +758,11 @@ def call_method(
             and args[0].is_python_constant()
             and isinstance(
                 self.obj,
```

Comment:
Why have this restriction?

Reply:
Good point! We should do that, we might need a method that we need to add to all variable trackers.